File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(long long n) {
    vector<int> answer;
    
    string numStr = to_string(n);
    for(int i = numStr.length() - 1; i>=0; --i)
    {
        answer.push_back(numStr[i] - '0');
    }
    
    return answer;
}

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.cs
using System;

public class Solution {
    public int[] solution(long n) {
        int[] answer = new int[] {};
        
        string str = n.ToString();
        int length = str.Length;
        Array.Resize(ref answer, length);
        
        for(int i = str.Length - 1; i >= 0; --i)
            answer[length - 1 - i] = str[i] - '0';
        
        return answer;
    }
}